dit/dit.py

```diff
@@ -572,10 +562,10 @@ def export(self, argv):
             file = open(output, 'w')
             fmt = output.split(".")[-1]
 
-        if fmt not in ['dit', 'org', 'json']:
+        if fmt not in ['dit', 'org']:
             raise Exception("Unrecognized format")
 
-        printer = __import__(fmt + 'printer')
+        printer = import_module('dit.' + fmt + 'printer')
```

Comment:
remove 'dit.' prefix, since this file is located at the same directory as the printer